test/spec/runners/bls.ex

```diff
@@ -76,8 +76,14 @@ defmodule BLSTestRunner do
     assert false
   end
 
-  defp assert_fast_aggregate_verify(_input, _output) do
-    assert false
+  defp assert_fast_aggregate_verify(
+         %{message: message, pubkeys: pubkeys, signature: signature},
+         output
+       ) do
+    case Bls.fast_aggregate_verify(pubkeys, message, signature) do
+      {:ok, confirmation} -> assert confirmation == output
+      {result, error_msg} -> assert result == :error, error_msg
+    end
```

Comment:
Since we only care about the _happy path_, this can be simplified using plain matches inside an assert. Example from the `assert_sign` function:
```elixir
assert {:ok, signature} = Bls.sign(private_key, message)
assert signature == output
```

Reply:
oh ok :+1:

Reply:
I tried, but there were four failed spec tests. 
For example, for `fast_aggregate_verify_tampered_signature_5e745ad0c6199a6c` the output is `false`  but the test fails with an `{:error, "BlstError(BLST_POINT_NOT_ON_CURVE)"}`. 
I think this error is generated by the `deserialize` methods used in the rust native part.

Reply:
My bad! Then we should check depending on the test's `output`. If the expected output is `false` we should check it returns either `{:ok, false}` or `{:error, reason}`, but if it's `true`, then it should always return `{:ok, true}`.

Reply:
maybe something  like this?:  
```
if output do
      assert {:ok, confirmation} = Bls.fast_aggregate_verify(pubkeys, message, signature)
      assert confirmation == output
else
  case Bls.fast_aggregate_verify(pubkeys, message, signature) do
    {:ok, failed_confirmation} -> assert failed_confirmation == false
    {result, error_msg} -> assert result == :error, error_msg
  end
end
```

Reply:
I think it'll be simpler if we keep the single `case`, but change the match arms:
```elixir
{:ok, true} -> assert output
{:ok, false} -> assert not output
{:error, reason} -> assert not output, reason
```

Reply:
fantastic, much clearer.

For the error case, would it be also necessary to check that is indeed an :error?
``` elixir
{result, reason} ->
  assert not output, reason
  assert result == :error
```

Reply:
No need, since we match against `{:error, reason}`, and not `{result, reason}`. Then if we try to match against a `{:non_error, "something"}`, for example, the test will fail because no match arm matched inside the `case`.